src/main/resources/application.properties

```diff
@@ -1,19 +1,19 @@
-## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
-spring.datasource.url=jdbc:mysql://localhost:3306/cooking_helper
-spring.datasource.username=root
-spring.datasource.password=move_ToExplore*21
-
-## Hibernate Properties
-# The SQL dialect makes Hibernate generate better SQL for the chosen database
-# spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
-
-#spring.jpa.generate-ddl=true
-#spring.jpa.hibernate.ddl-auto=update
-spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQLDialect
-#spring.jpa.show-sql=true
-
-## This is important
-# Hibernate ddl auto (create, create-drop, validate, update)
-#spring.jpa.hibernate.ddl-auto = validate
-# logging.level.root=debug
-spring.flyway.baseline-on-migrate=true
+## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
+spring.datasource.url=jdbc:mysql://localhost:3306/cooking_helper
+spring.datasource.username=root
+spring.datasource.password=move_ToExplore*21

```

Comment:
Passwords should not be visible in git repository. You may have also received github's security warning email.
You can pass passwords as environment variables, like
`spring.datasource.password=${DB_PASSWORD}`
and run application with parameters `DB_PASSWORD=move_ToExplore*21`

https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html